ui/jobs/jobs.js

```diff
@@ -2491,6 +2499,105 @@ class Bars {
   }
 
   setupBrd() {
+    // DoT
+    const causticBiteBox = this.addProcBox({
+      id: 'brd-procs-causticbite',
+      fgColor: 'brd-color-causticbite',
+    });
+    const stormbBiteBox = this.addProcBox({
+      id: 'brd-procs-stormbite',
+      fgColor: 'brd-color-stormbite',
+    });
+    [
+      EffectId.Stormbite,
+      EffectId.Windbite,
+    ].forEach((effect) => {
+      this.mobGainEffectFromYouFuncMap[effect] = () => {
+        stormbBiteBox.duration = 0;
+        stormbBiteBox.duration = 30 - 0.5;
```

Comment:
Can you leave a comment about what the `0.5` is for? I assume it's something like animation delay?

Reply:
For an unknown reason, the effect duration of these two DoT is not actually 30s, but 29.5s.
I think maybe it is because countdown starts before DoT appear on target.